tests/components/deconz/test_binary_sensor.py

```diff
@@ -34,204 +35,508 @@ async def test_no_binary_sensors(hass, aioclient_mock):
     assert len(hass.states.async_all()) == 0
 
 
-async def test_binary_sensors(hass, aioclient_mock, mock_deconz_websocket):
-    """Test successful creation of binary sensor entities."""
-    data = {
-        "sensors": {
-            "1": {
-                "name": "Presence sensor",
-                "type": "ZHAPresence",
-                "state": {"dark": False, "presence": False},
-                "config": {"on": True, "reachable": True, "temperature": 10},
-                "uniqueid": "00:00:00:00:00:00:00:00-00",
+binary_sensor_test_data = [
```

Comment:
Please define constants with all caps to follow PEP8.

Reply:
Resolved